README.md

```diff
@@ -26,11 +27,19 @@ Service to backup and/or restore mysql databases to/from S3
 
 >**It's recommended that your S3 bucket have versioning turned on.**
 
+`B2_APPLICATION_KEY_ID` (optional) Backblaze application key ID
+
+`B2_APPLICATION_KEY` (optional) Backblaze application key secret
+
+`B2_BUCKET` (optional) Name of the Backblaze B2 bucket, e.g., _database-backups_. When `B2_BUCKET` is defined, the backup file is copied to the B2 bucket.
+
+>**It's recommended that your B2 bucket have versioning and encryption turned on.**
```

Comment:
Is it worth explaining why these are recommended?

Reply:
Perhaps.  I was following the practice started by the description of `S3_BUCKET`.  If versioning isn't turned on, you only have one backup as each copy to S3 and B2 overwrites the existing backup file (they have the same name).  Encryption sounded like a "good thing" and has the side effect of preventing downloads from the Backblaze GUI (you have to install and use the `b2` command).  Do you think expanding the description is desirable?

Reply:
I do, but maybe just a personal thing. I often like to know the reason why, if it can be summarized briefly. Please don't feel compelled though, if you don't want to.